### ISSUE TITLE
Icon request: fa-Raft or kayak

### ISSUE THREAD

**Issue by DigitalAlan** [FortAwesome/Font-Awesome#10772]
I see a boat but looking for a river raft or person in a kayak.

**Comment by DigitalAlan** [comment]
![image](https://cloud.githubusercontent.com/assets/12120023/23764697/75c13c6e-04cc-11e7-89d3-225a7d93f1bd.png)

**Comment by DigitalAlan** [comment]
![image](https://cloud.githubusercontent.com/assets/12120023/23764738/9962ad42-04cc-11e7-93c6-9b94effe0598.png)

**Comment by DigitalAlan** [comment]
![image](https://cloud.githubusercontent.com/assets/12120023/23764763/b43ca168-04cc-11e7-9b8a-dc469cca7b52.png)

![image](https://cloud.githubusercontent.com/assets/12120023/23764781/bcf5db12-04cc-11e7-9b0f-172bd7f37b49.png)

**Comment by h0lg** [comment]
yeah, a kayak or canoe or paddles would be nice! See #11014 and #13240 as well.

**Comment by greggbanse** [comment]
I see mention of a kayak / kayaker but I'm not sure it's under consideration so...

**Icon request: sit in kayak**

SIt in kayaks (used for ocean and moving water) are different than sit on top (used for fishing and flat water)

https://www.shutterstock.com/image-vector/canoe-flatwater-isolated-vector-silhouette-ink-2301583121 (ignore the file name - it's actually an image of a sit in kayak)

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
